src/rust/db/examples/seed_db.rs

```diff
@@ -58,6 +58,21 @@ fn place_random_orders(
             },
         )
         .unwrap();
+
+        add_taker_event(
+            conn,
+            &NewTakerEvent {
+                market_id: market_id_bd,
+                side,
+                market_order_id: &id.into(),
+                maker: "0x123",
+                custodian_id: None,
+                size: &BigDecimal::from_u64(size >> 2).unwrap(),
+                price: &BigDecimal::from_u64(price).unwrap(),
+                time: Utc::now(),
+            },
+        )
+        .unwrap();
```

Comment:
It looks like we might deprecate taker events in lieu of fill events per #315

Reply:
Yup. For now let me just remove this file from this PR.